docs/configuration.asciidoc

```diff
@@ -876,6 +876,9 @@ https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?#allowsynchronou
 because a large number of blocking synchronous I/O operations can lead to thread pool starvation,
 which makes the application unresponsive. If your application becomes unresponsive with this
 feature enabled, consider disabling capturing.
+
+In ASP.NET (.NET Full Framework), this setting has no effect in case of non-buffered requests
```

Comment:
@gregkalapos as discussed in the PR walkthrough I added a note in the documentation about the "bufferless limitation".

Reply:
I like that, thanks!